Source/RageCommunity.Library/Wrappers/NativeTaskWrappers.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using Rage;
+using Rage.Native;
+
+namespace RageCommunity.Library.Wrappers
+{
+    public static partial class NativeWrappers
+    {
+        /// <summary>
+        /// Causes the given ped to start playing the specified scenario.  A list of scenarios can be found <a href="https://github.com/DurtyFree/gta-v-data-dumps/blob/master/scenariosCompact.json">here</a>.
+        /// </summary>
+        public static void TaskStartScenarioInPlace(Ped ped, string scenarioName, int unkDelay = 0, bool playEnterAnimation = true)
+        {
+            NativeFunction.Natives.TASK_START_SCENARIO_IN_PLACE(ped, scenarioName, 0, playEnterAnimation);
```

Comment:
From [NativeDB](https://nativedb.dotindustries.dev/natives?search=task_start_scenario_in_place):
> unkDelay - Usually 0 or -1, doesn't seem to have any effect. Might be a delay between sequences.

Not sure if the parameter should be removed or kept so people can try to modify it if they want.

Reply:
We should just pass the parameter to the native, maybe someone figures out what it is for.

Reply:
Resolved